content/influxdb/v2.0/reference/cli/influx/bucket/create.md

```diff
@@ -83,3 +90,15 @@ influx bucket create \
   --retention 30d \
   --shard-group-duration 2d
 ```
+
```

Comment:
May be helpful in the task-based "[Create a bucket](https://docs.influxdata.com/influxdb/cloud/organizations/buckets/create-bucket/)" doc to add link to how to create a bucket with explicit schema?